### DISCUSSION TITLE
Problema con integrazione Tuya

### DISCUSSION THREAD

**Discussion by shra1976** [indomus/forum#1612] [HUB: Home Assistant]
Ciao a tutti!
Vi spiego il mio problema sperando che qualcuno possa aiutarmi.
Per comandare le tapparelle uso un dispositivo Zigbee Tuya. 
Con i pulsanti fisici e con l'applicazione SmartLife tutto funziona bene.
Su Home Assistant l'integrazione mi funziona ma con la posizione invertita.
Mi spiego meglio: quando la tapparella è completamente aperta su HA la vedo in posizione 0 e quindi come se fosse completamente chiusa e mi viene inibita la freccia verso il basso. Quando è chiusa la vedo in posizione 100 e mi viene inibita la freccia verso l'alto (quindi non posso più muoverla).
Come faccio a invertire le posizioni?
Grazie,
Marco.

**Comment by 50494554524F** [comment]
usa "template cover" ed inverti il funzionamento...

**Reply by shra1976** [reply]
grazie del suggerimento!
